### DISCUSSION TITLE
EMANE Event Monitor

### DISCUSSION THREAD

**Discussion by 1-Jone** [coreemu/core#725] [Q&A]
Recently, I tried to reproduce the demo of [EMANE Emulation Event Log (EEL) Generator].   I prepared eventservice.xml, eelgenerator.xml and scenario.eel.    Where should I put these xml files and eel files?   How do I run these files in core-gui?

**Comment by bharnden** [comment]
The directions are all here: http://coreemu.github.io/core/emane/eel.html

They are placed on the host, with the proper configuration of the control network device, as noted.

**Comment by 1-Jone** [comment]
Thanks for your answer. 
When I run emaneeventservice eventservice.xml -l 3, I am facing the problem . 

12:06:02.676839 ABORT StartException: Platform Event Service: Unable to open Event Service socket: '224.1.2.8:45703'.

Possible reason(s):
 * No Multicast device specified and routing table non deterministic
   (no multicast route and no default route).
 * Multicast device ctrl0 does not exist or is not up.
ioctl SIOCGIFADDR: No such device

May I ask how to solve it?

**Comment by bharnden** [comment]
On the page I linked:

> ***WARNING***: make sure to set the eventservicedevice to the proper control network value

**Comment by 1-Jone** [comment]
What directory do I need to put eventservice.xml, eelgenerator.xml and scenario.eel?

**Comment by bharnden** [comment]
They need to be placed in the same directory, assuming you are copying the examples as they assume relative path. Otherwise you can provide an absolute path and place it where you like.

What directory is irrelevant, just that you are using the command and they can find all the associated files, as noted in the example.

**Comment by 1-Jone** [comment]
WARNING: make sure to set the eventservicedevice to the proper control network value
Where do I get the value of this control network?
Is this value the ctrl0 of the eventservicedevice in the emane settings?

**Comment by bharnden** [comment]
That would be the name of the device on the host, that is tied to the control network used by the nodes.

**Comment by 1-Jone** [comment]
Thanks for the answer. How should I get this value?

**Comment by bharnden** [comment]
`ip a` will list the devices on your host. Use the name of the one on the same subnet, as your nodes control network.
